BREAKING.md

```diff
@@ -0,0 +1,21 @@
+# Breaking Changes
+
+This is a history of NIP changes that potentially break pre-existing implementations, in
+reverse chronological order.
+
+| Date        | Commit    | NIP      | Change |
+| ----------- | --------- | -------- | ------ |
+| 2024-02-15  | [afbb8dd0](https://github.com/nostr-protocol/nips/commit/afbb8dd0) | [NIP-39](39.md) | PGP identity was removed |
+| 2024-02-07  | [d3dad114](https://github.com/nostr-protocol/nips/commit/d3dad114)  | [NIP-46](46.md) | Connection token format was changed |
+| 2024-01-30  | [1a2b21b6](https://github.com/nostr-protocol/nips/commit/1a2b21b6)  | [NIP-59](59.md) | 'p' tag became optional |
+| 2023-01-27  | [c2f34817](https://github.com/nostr-protocol/nips/commit/c2f34817)  | [NIP-47](47.md) | optional expiration tag should be honored |
+| 2024-01-10  | [3d8652ea](https://github.com/nostr-protocol/nips/commit/3d8652ea)  | [NIP-02](02.md) | list entries should be chronological |
```

Comment:
how is this order breaking things? Before there was no order. And it's a SHOULD, not a MUST.

Reply:
> I would rather have false positives than false negatives.

Maybe we need a rating system for how bad it is based on how many clients were/are expected to be broken. 

Having a massive list of mostly minor things will not be that helpful for the average dev. We should focus on the changes that broke everything and keep a list of the minor stuff on the side.

Reply:
Could use 'breaking' and 'advisory'.  Don't want to be too complex.